openc3/spec/packets/parsers/xtce_parser_spec.rb

```diff
@@ -102,6 +102,9 @@ def add_tlm_packet_item(tf, packet, item)
       tf.puts "    <xtce:EntryList/>"
       tf.puts "    <xtce:BaseContainer containerRef=\"#{packet}_Base\"/>"
       tf.puts "  </xtce:SequenceContainer>"
+      tf.puts "  <xtce:SequenceContainer name=\"#{packet}_BY_REF\" shortDescription=\"Telemetry\">"
+      tf.puts "    <xtce:ContainerRefEntry containerRef=\"#{packet}_Base\"/>"
+      tf.puts "  </xtce:SequenceContainer>"
```

Comment:
How is `ContainerRefEntry` different from `BaseContainer` with a containerRef?

Reply:
You could theoretically have multiple ContaineRefEntry in the same packet.